scripts/config.pl

```diff
@@ -96,7 +98,7 @@
 
 # Things that should be enabled in "full" even if they match @excluded
 my @non_excluded = qw(
-PLATFORM_[A-Z0-9]+_ALT
+PLATFORM_[_A-Z0-9]+_ALT
```

Comment:
This now matches double (or more) `_` like `PLATFORM_____WHATEVER_STUFF_____ALT`. Consider removing all `_` from outside the `[]`.

Reply:
I personally don't think this matters. Whether the underscore is inside or outside the brackets, they'll still match to multiple underscores, but I don't think that's an issue.